pkg/ibmvpc/parser.go

```diff
@@ -30,12 +30,51 @@ type IBMresourcesContainer struct {
 	datamodel.ResourcesContainerModel
 }
 
-func NewIBMresourcesContainer(rc common.ResourcesContainerInf) (*IBMresourcesContainer, error) {
+// NewIBMresourcesContainer is used to return empty IBMresourcesContainer and also initialize
+// vpcmodel.NetworkAddressLists with ibm Public internet and service network
+// if you do not use this function, you need to initialize vpcmodel.NetworkAddressLists
+func NewIBMresourcesContainer() *IBMresourcesContainer {
+	vpcmodel.InitNetworkAddressLists(GetPublicInternetAddressList(), GetServiceNetworkAddressList())
+	return &IBMresourcesContainer{}
+}
+
+// IBM All public IP addresses belong to one of the following public IP address ranges:
+func GetPublicInternetAddressList() []string {
```

Comment:
Does this function need to be exported?
Does it have to be a function? Can it be a global variable or a local variable inside `NewIBMresourcesContainer()`?